website/docs/artist_inbox.md

```diff
@@ -53,3 +61,16 @@ Desktop notifications and notification sounds work independently to one another.
 :::warning
 Desktop notifications will not work over HTTP connections due to strict browser security measures.
 :::
+
+## Importance Table
```

Comment:
I feel like this table should be higher up above `# Realtime Notifications`?
Because it relates to Watchers, not to realtime notifications?